Goutte/Client.php

```diff
@@ -129,14 +129,16 @@ protected function doRequest($request)
                 $requestOptions['body'] = $content;
             } else {
                 if ($files = $request->getFiles()) {
-                    $requestOptions['multipart'] = [];
+                    $requestOptions['multipart'] = array();
```

Comment:
I add those changes to pass the coding standard test. The only change needed is in the lines 141-143